Gopkg.lock

```diff
@@ -660,7 +660,6 @@
   analyzer-version = 1
   input-imports = [
     "github.com/GeertJohan/go.rice",
-    "github.com/GeertJohan/go.rice/embedded",
```

Comment:
I know this file is autogenerated, but how come rice/embedded dependency is not in the lock file anymore ?

Reply:
What bothers me is that the Gopkg.lock is modified, but not the Gopkg.toml, this should not happen